gradle/wrapper/gradle-wrapper.properties

```diff
@@ -0,0 +1,6 @@
+#Sat Mar 21 14:50:43 PDT 2020
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.8.1-all.zip
```

Comment:
Is this supposed to be ‘https\://‘ for formatting or syntax reasons?

Reply:
This was auto generated and seemed to work? Also, `:` can be a special character (I think in bash among others)